Sources/App/Extensions/String+AbsolutePath.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  File.swift
+//  
+//
+//  Created by Julian Gentges on 10.08.20.
+//
+
+import Foundation
+
+extension String {
+    public var absolutePath: String {
```

Comment:
I believe that this function is not about building an absolute path, it just appends a trailing slash to a sting if it has no suffix. I don't really think that we need this extension to make things work for now.

Reply:
I added this function to prevent user error when reading the path from the environment. It happened to me that I accidentally forgot the trailing slash in the end of the path. The framework than just runs until it crashes because there are files missing. But it also creates a new folder below your working-directory.
I can understand if you don't like this specific solution, but I would suggest to prevent that user error or throw an actual error beforehand.

Reply:
You are right about paths, maybe we should extend the Environment object with a static path() loader which could perform this job, also that could expand the tilde if necessary. What do you think? 

I'm going to merge this PR and we can open a new one for this issue if needed